### PR TITLE
Copyright notice

### PR DESCRIPTION
**Pull request by matthiasbeyer**
I guess this is a good idea, isn't it.

@neithernut what do you think about the text? Is it appropriate?

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
And I have a typo. Fix incoming.

**Comment by matthiasbeyer** [comment]
And I marked this hard-required as I want this in 0.2.0 included.

**Comment by neithernut** [comment]
Mh... are you using a standard copyright notice template? I don't recognize it.

**Comment by neithernut** [comment]
Also, I'd have to have a look at the LGPL again. Some licenses only require a notice, others want it in a specific form, yet others (short ones) want themselves copied to every single file.

**Comment by matthiasbeyer** [comment]
> Mh... are you using a standard copyright notice template? I don't recognize it.

Nope.

> Also, I'd have to have a look at the LGPL again. Some licenses only require a notice, others want it in a specific form, yet others (short ones) want themselves copied to every single file.

Good point.

**Comment by neithernut** [comment]
Right at the beginning of the actual license text, it says:

>  This License Agreement applies to any software library or other
program which contains a notice placed by the copyright holder or
other authorized party saying it may be distributed under the terms of
this Lesser General Public License (also called "this License").

We should have added a notice/that we should add a notice as soon as possible. The `LICENSE` file also contains a section *after* the terms and conditions, containing some recommendations regarding the notice:

> To apply these terms, attach the following notices to the library.  It is
safest to attach them to the start of each source file to most effectively
convey the exclusion of warranty; and each file should have at least the
"copyright" line and a pointer to where the full notice is found.
>
> [...]

You should just copy that template, fill out the fields and prepend it to every single source file.

**Comment by matthiasbeyer** [comment]
@neithernut What do you think?

**Comment by neithernut** [comment]
Mh... I'm not sure about the `... and contributors` part in the copyright info. Normally, one would list all the contributors (one per line) or one abstract entity (for projects steered by foundations). In the latter case, you'd have to "collect" the copyright from all the contributors.

Maybe you should have a look at other projects using the LGPL.

**Comment by matthiasbeyer** [comment]
I'm pretty sure it doesn't matter. Listing all contributors would be way too verbose IMHO.

As we do not relicense, I'm pretty sure I do not have to ask people for acknowledgement here.

So I'd say "and contributors" or nothing :smile:

**Comment by neithernut** [comment]
The copyright notice is, in a way, a separate concern from the license. So, it still can bite you.

For now, `and contributors` might be OK. However, maybe you'll be forced to change the copyright notice at some point.

**Comment by matthiasbeyer** [comment]
@neithernut is this a 

> TL;DR: Ack?

? :smile:

**Comment by neithernut** [comment]
It's more a disclaimer than an ACK.

**Comment by matthiasbeyer** [comment]
If there are no more points left, I'll merge it now.

**Comment by neithernut** [comment]
No more objections from me.
